Dev-bundle is failing after mvn clean + clean-frontend

### Description of the bug

I'm building an application with a third party addon viritin.
A new dev-bundle is created when I run the application.
If I run `mvn clean vaadin:clean-frontend`, then the application does not start anymore.

> 
> Caused by: java.lang.IllegalStateException:
> 
> Failed to find the following css files in the `node_modules` or `/Users/xxxx/hackathon-24-1/./frontend` directory tree:
> - lumo-css-framework/all-classes.css
> Check that they exist or are installed. If you use a custom directory for your resource files instead of the default `frontend` folder then make sure it's correctly configured (e.g. set 'vaadin.frontend.frontend.folder' property)
> 

### Expected behavior

The application should run.

I think it's related to the CSS imported in the addon from a NPM package. Since I can't reproduce it with the other addons I tested.

### Minimal reproducible example

https://github.com/vaadin/hackathon-24-1/tree/dev-bundle-tests



### Versions

- Vaadin / Flow version: Vaadin 24.1.beta2 and 24.0.5
- Java version: 17

--- a/flow-server/src/main/java/com/vaadin/flow/server/frontend/AbstractUpdateImports.java
+++ b/flow-server/src/main/java/com/vaadin/flow/server/frontend/AbstractUpdateImports.java
@@ -654,7 +654,7 @@ abstract class AbstractUpdateImports implements Runnable {
         } else {
             addLines(lines, String.format(INJECT_CSS, i, cssImport));
         }
-        return found;
+        return found || !options.isBundleBuild();
     }
 
     private String notFoundMessage(Set<String> files, String prefix,

